src/App.js

```diff
@@ -13,6 +13,22 @@ function App() {
     JSON.parse(localStorage.getItem('tokenList')) || [],
   );
 
+  // add token to local storage
+  const addTokenToLocalStorage = (token) => {
```

Comment:
very clear functions, I know what they're doing just from the title

I keep trying to picture what the stringify and parse are doing -- are they turning the whole array into a string or each item in the array to a string? And was there a bug happening that directed you to this solution?